### PR TITLE
Bugfix: Add optional prop for include ' | TypeHero' in OG image

### PR DESCRIPTION
**Pull request by maxdemaio**
## Description
Strips the OG image h1 title to not have ` | TypeHero` 👍🏻 

## Related Issue
#671 

## Motivation and Context
For a more clear dynamically generated og image 

## How Has This Been Tested?
Locally and checking the OG image link in the `head` of the HTML

## Screenshots/Video (if applicable):

Before:
image

After:
image

### CODE REVIEW

File: apps/web/src/app/metadata.ts [hunk 1]
```diff
@@ -86,6 +86,7 @@ export const buildMetaForChallenge = async ({
   username,
   difficulty,
   date,
+  removeThTitle,
```
Comment (Hacksore) [inline apps/web/src/app/metadata.ts:89]: Dead quode

**Review by Hacksore — CHANGES_REQUESTED** [review]

**Review by Hacksore — APPROVED** [review]

File: apps/web/src/app/metadata.ts (hunk 1 above)
Comment (maxdemaio) [reply apps/web/src/app/metadata.ts]: I was still cooking

Comment (Hacksore) [reply apps/web/src/app/metadata.ts]: image

Comment (maxdemaio) [reply apps/web/src/app/metadata.ts]: > [!IMPORTANT]  
> By cooking I mean fixing my dead code

**Pull request merged**